ocs_ci/helpers/helpers.py

```
@@ -7874,3 +7875,220 @@ def verify_file_ownership(pod_obj, file_path, expected_uid, expected_gid):
+
+    cluster_name = "ibm-spectrum-scale"
+
+    # 1. Apply MCO (Operator) using pinned commit URL
+    mco_url = (
+        "https://raw.githubusercontent.com/IBM/ibm-spectrum-scale-container-native/"
+        "502fbcbe968651d2c94bd6b8419958c835383696/generated/scale/mco/mco.yaml"
+    )
+    try:
+        exec_cmd(f"oc apply -f {mco_url}")
+    except CommandFailed as e:
+        raise CommandFailed(f"Scale operator deployment failed: {e}") from e
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)